xclim/sdba/detrending.py

```diff
@@ -77,8 +89,43 @@ def retrend(self, da: xr.DataArray):
         trend = self.get_trend(da)
         return self._retrend(da, trend)
 
-    def _set_fitds(self, ds):
-        self.fit_ds = ds
+    def save_fit(
```

Comment:
Same comment as above. If the tmp file is meant to be deleted, should it really be in the public API ?

Reply:
Same answer as above. For medium and small size cases, this is not beneficial, thus why giving the option as a public method.